region/maridia/inner-pink/Halfie Climb Room.json

```diff
@@ -2298,6 +2329,49 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [2, 3],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "requires": [
+        {"obstaclesCleared": ["R-Mode"]},
+        "Gravity",
+        "Morph",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"or": [
+              {"resourceMissingAtMost": [{"type": "Missile", "count": 2}]},
+              {"resourceMissingAtMost": [{"type": "Super", "count": 1}]}
+            ]},
+            {"or": [
+              {"and": [
+                {"resourceMissingAtMost": [{"type": "Missile", "count": 0}]},
```

Comment:
Leave the lower two mochtroid farm and remove the movement item stuff.
Having a way up will be included already with the [1, 3] strat.